docs/content/deployment/guides/kubernetes/customizing-your-deployment.mdx

```diff
@@ -136,9 +210,11 @@ def my_job():
     my_op()
 ```
 
-Non-k8s run launchers and executors will ignore the `dagster-k8s/config` tag.
+Other executors will ignore the `dagster-k8s/config` tag when it is set on an op or asset.
+
+### Precedence rules
```

Comment:
Thoughts on taking the opportunity to reformat the second part of this section? See attached for what I had in mind. I had a bit of trouble reading and comparing the two situations here - I feel like this formatting approach could make that easier for folks.

I know this wasn't your change, but I think it's a bit more obvious now that it's in a section with only one other paragraph:

![Screenshot 2023-08-15 at 4 47 16 PM](https://github.com/dagster-io/dagster/assets/16615212/d197d5bf-b9af-46b2-9be8-5dfb7f30247f)

And the code if you feel fine with this:

````
### Precedence rules

If a Kubernetes configuration dictionary (like `container_config`) is specified at both the instance level in the Helm chart and in a specific Dagster job or op, the dictionaries will be shallowly merged. The more specific configuration takes precedence if the same key is set in both dictionaries.

Consider the following example:

- **In the Helm chart**, `k8sRunLauncher.runK8sConfig.podSpecConfig` is set to:

   ```json
   {"nodeSelector": {"disktype": "ssd"}, "dns_policy": "ClusterFirst"}
   ```

- **But a specific job** has the `pod_spec_config` key in the `dagster-k8s/config` tag set to:

   ```json
   {"nodeSelector": {"region": "east"}}
   ```

Then the node selector from the job and the DNS policy from the Helm chart will be applied, since only the node selector is overridden in the job.
````

Reply:
I do feel fine with this, thanks!